### PR TITLE
Update/plateoutfixes

### PR DESCRIPTION
**Pull request by x-anderson**

### CODE REVIEW

File: an/Liquid_handling/PlateOut/PlateOut.an [hunk 1]
```diff
@@ -46,78 +46,91 @@ Setup {
...
+	var warnings []string
 
 	//set counter variable to count up number of wells used, and set at the number of wells already used in the output agar plate (Default = 0)
 	var counter int = WellsAlreadyUsed
 
-	//set up error variable
-	var err error
+	if WellsAlreadyUsed < 0 {
+			WellsAlreadyUsed = defaultWellsAlreadyUsed
+			wellsUsedError := fmt.Errorf("Invalid WellsAlreadyUsed specified, assinging to default: %b", defaultWellsAlreadyUsed)
```
Comment (chr1sgrant) [inline an/Liquid_handling/PlateOut/PlateOut.an]: %d ?

File: an/Liquid_handling/PlateOut/PlateOut.an [hunk 2]
```diff
@@ -46,78 +46,91 @@ Setup {
...
+			warnings = append(warnings, wellsUsedError.Error())
+			}
 
-	//set up a slice to add the plate out reactions to
-	var plateOutVolumes []*wtype.LHComponent
+	if AgarPlateNumber <= 0 {
+			AgarPlateNumber = defaultAgarPlateNumber
+			agarPlateNumError := fmt.Errorf("Invalid AgarPlateNumber specified, assinging to default: %b", defaultAgarPlateNumber)
```
Comment (chr1sgrant) [inline an/Liquid_handling/PlateOut/PlateOut.an]: does the %b return the error correctly? 
I think %d is the right choice??
https://golang.org/pkg/fmt/

**Comment by chr1sgrant** [comment]
You need to update the bundle files to not reference the parameters you removed (i.e. Dilution)

**Comment by x-anderson** [comment]
I have updated the bundle files to remove the Dilution/Diluent parameters - can you see those edits (when i look at the file comparison i can see them)

**Comment by chr1sgrant** [comment]
@synthacebot test this please

**Comment by chr1sgrant** [comment]
It's failing the tests:
```bash
	main_test.go:86: error running workflow "/home/travis/gopath/src/github.com/antha-lang/elements/workflows/examples/PlateOut/workflow_Auto.json" with parameters "/home/travis/gopath/src/github.com/antha-lang/elements/workflows/examples/PlateOut/parameters_Auto.yml": cannot assign parameter "Diluent" of process "AutoPlateOut" to "LB": unknown parameter
```

**Comment by x-anderson** [comment]
Hmm....it seems to be trying to simulate with files with `.Auto` (capitalised) instead of `.auto`, but the committed and pushed files were all `.auto`.  Ive verified this, and re-pushed those lower-case files so unsure why still failing. Ive went back through all the commits and all the parameters and workflow files are fine. Im unsure how to resolve this?

**Pull request closed without merging**